libs/libcommon/src/libcommon/config.py

```diff
@@ -321,6 +338,15 @@ class ProcessingGraphConfig:
                 "triggered_by": ["dataset-config-names", "config-opt-in-out-urls-count"],
                 "job_runner_version": PROCESSING_STEP_DATASET_OPT_IN_OUT_URLS_COUNT_VERSION,
             },
+            "split-duckdb-index": {
+                "input_type": "split",
+                "triggered_by": [
+                    "split-first-rows-from-streaming",
+                    "split-first-rows-from-parquet",
```

Comment:
I changed to use parquet instead of first-rows, but kept split level as per the comment https://github.com/huggingface/datasets-server/pull/1296#issuecomment-1578855161